public/index.js

```diff
@@ -1,34 +1,29 @@
 (function() {
 
-  // const message = document.getElementsByClassName('message')[0];
-  // const red = document.getElementsByClassName('red')[0];
-  // const blue = document.getElementsByClassName('blue')[0];
-  // const yellow = document.getElementsByClassName('yellow')[0];
-
+  const redBtn = document.getElementById('red-button');
+  const blueBtn = document.getElementById('blue-button');
+  const yellowBtn = document.getElementById('yellow-button');
   const body = document.querySelector('body');
 
-  body.style.backgroundColor = "cyan";
```

Comment:
Make sure to test the functionality of this file to make sure it works when you trial this on your own.